util/Migrator/DbScripts/2021-04-27_00_OrganizationUserDataType.sql

```diff
@@ -0,0 +1,26 @@
+-- Create OrganizationUser Type
```

Comment:
Would be great to combine all of these in a single update script to reduce the sheer number of DbScripts in the migrator, especially since they're all inter-connected with the corresponding server C# code changes.

Reply:
Not sure what my problem was before. I was able to combine these and batch delete no problem :shrug: